rustfmt.toml

```diff
@@ -4,17 +4,3 @@ use_field_init_shorthand = true
 use_small_heuristics = "Max"
 use_try_shorthand = true
 max_width = 100
-
-# Unstable features below
```

Comment:
Doesn't this unnecessarily increase the changed-line count of this PR? Can this be handled in a separate PR to make review easier?

Reply:
Were tmp changes that I forgot to undo